### ISSUE TITLE
Wrong declaration for ped muni 

### ISSUE THREAD

**Issue by Olovorr** [Witiko/fithesis3#10]
Hello guys, 

according to dean in ped muni, declaration should be in this wording -

`
„Prohlašuji, že jsem bakalářskou práci vypracovala samostatně, s využitím pouze citovaných pramenů, dalších informací a zdrojů v souladu s Disciplinárním řádem pro studenty Pedagogické fakulty Masarykovy univerzity a se zákonem č. 121/2000 Sb., o právu autorském, o právech souvisejících s právem autorským a o změně některých zákonů (autorský zákon), ve znění pozdějších předpisů.“
`

I already have PR ready, but I don't have neccessary access rights. 

If someone could fix it / give me access right so I can create PR, I would be really glad. 👼

**Comment by Witiko** [comment]
Hey,

thank you for the issue report. You should have all the rights necessary to make a pull request.

**Comment by Olovorr** [comment]
Thanks for the fast response!

Unfortunately I am unable to create the upstream branch 😕 

<img width="968" alt="screen shot 2017-10-21 at 20 13 36" src="https://user-images.githubusercontent.com/15366622/31854614-6118ad06-b69c-11e7-92ba-dc03246c3e1e.png">

**Comment by Witiko** [comment]
You need to create a copy of (fork) the repository, commit your changes to the copy of the repository, and then issue a pull request through GitHub (see [the corresponding page in the GitHub documentation][1]).

[1]: https://help.github.com/articles/creating-a-pull-request/

**Comment by Witiko** [comment]
At the time of writing this message, you need to:
1. hit the `Fork` button in the upper right corner of this page,
2. add your forked repository as a new remote:
    ```
    git remote add my-fork git@github.com:Olovorr/fithesis3.git
    ```
3. push your changes to your forked repository:
    ```
    git push my-fork fix/ped-declaration
    ```
4. navigate to <https://github.com/Olovorr/fithesis3>, switch to the `fix/ped-declaration` branch and click the `Create pull request` button.

If you don't want to bother, I can just add your contribution and mention you in the commit message.

**Comment by Olovorr** [comment]
Sorry for the delay, I was out on the conference so I could not get around to do it before today 🙂. 

Thanks for explaining! So far I never used fork as a method for PR, I always set master as a protected branch and every new contribution had to go through PR from other branch, therefore no fork was necessary, so this was new to me. Glad that I learned something new though 🙂 

Also if there is any issue with PR, just write comment and I will update it accordingly - I am just not that familiar with used syntax so maybe I did something wrong.
